addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -62,3 +62,86 @@ Switch APCRunning { channel="exec:command:apc:run"}
 Number APCExitValue {channel="exec:command:apc:exit"}
 DateTime APCLastExecution {channel="exec:command:apc:lastexecution"}
 ```
+
+## Full Example
+
+Following is a example shown how to read out the temperature of the RPI with all files needed to set up and debug or log the executed command.
+The folder structure is for an installation with apt-get, for a different installation setup look [here](http://docs.openhab.org/installation/linux.html#file-locations) or search for them in your drive.
+
+First we need to check if the user openhab is able to execute the command we want to execute. The first command is executet as the user we are logged in the second as user openhab. If you get an error or an massages which indicates the command does not execute properly as user openhab then try to google the needed permissions and set them for the user openhab. More details [here](https://community.openhab.org/t/2-openhab2-rpi-system-temperature-chart-with-persistence/35182) and [here](https://community.openhab.org/t/openhab-sudo-exec-binding/34988).
```

Comment:
Typo 'executet' => executed, 'an massages' a message, maybe the third sentence could profit from some comma or other rewriting. Maybe you can also clarify the here so it's clear where the link is going to before the user clicks it, something like 'on a community thread about missing right to do x'